modules/utils/src/test/java/io/uhndata/cards/serialize/internal/BareProcessorTest.java

```diff
@@ -0,0 +1,330 @@
+
+        Property property = node.getProperty(QUESTIONNAIRE_PROPERTY);
+        JsonString input = Json.createValue(property.getString());
+        JsonValue jsonValue = this.bareProcessor.processProperty(node, property, input, mock(Function.class));
+        assertNotNull(jsonValue);
+        assertEquals(input, jsonValue);
+    }
+
+    @Test
+    public void processPropertyForJcrPropertyReturnsNull() throws RepositoryException
+    {
+        Node node = this.context.resourceResolver().getResource(TEST_FORM_PATH).adaptTo(Node.class);
+
+        Property property = node.getProperty(NODE_TYPE);
+        JsonValue jsonValue = this.bareProcessor.processProperty(node, property, Json.createValue(property.getString()),
+                mock(Function.class));
+        assertNull(jsonValue);
+    }
+
+    @Test
+    public void processPropertyForSlingPropertyReturnsNull() throws RepositoryException
+    {
+        Node node = this.context.resourceResolver().getResource(TEST_FORM_PATH).adaptTo(Node.class);
+
+        Property property = node.getProperty(RESOURCE_TYPE);
+        JsonValue jsonValue = this.bareProcessor.processProperty(node, property, Json.createValue(property.getString()),
+                mock(Function.class));
+        assertNull(jsonValue);
+    }
+
+    @Test
+    public void processChildForJcrChildReturnsNull() throws RepositoryException
+    {
+        Node node = this.context.resourceResolver().getResource(TEST_FORM_PATH).adaptTo(Node.class);
+        Node child = mock(Node.class);
+        when(child.getName()).thenReturn("jcr:name");
+
+        JsonValue jsonValue = this.bareProcessor.processChild(node, child, mock(JsonValue.class), mock(Function.class));
+        assertNull(jsonValue);
+    }
+
+    @Test
+    public void processChildCatchesRepositoryExceptionReturnsInputValue() throws RepositoryException
+    {
+        Node node = this.context.resourceResolver().getResource(TEST_FORM_PATH).adaptTo(Node.class);
+        Node child = mock(Node.class);
+        when(child.getName()).thenThrow(new RepositoryException());
+
+        JsonValue input = mock(JsonValue.class);
+        JsonValue jsonValue = this.bareProcessor.processChild(node, child, input, mock(Function.class));
+        assertNotNull(jsonValue);
+        assertEquals(input, jsonValue);
+    }
+
+    @Test
+    public void processChildForAnswerChildReturnsInputValue() throws RepositoryException
+    {
+        Session session = this.context.resourceResolver().adaptTo(Session.class);
+        Node node = session.getNode(TEST_FORM_PATH);
+        Node child = session.getNode(TEST_FORM_PATH + "/a1");
+
+        JsonValue input = mock(JsonValue.class);
+        JsonValue jsonValue = this.bareProcessor.processChild(node, child, input, mock(Function.class));
+        assertNotNull(jsonValue);
+        assertEquals(input, jsonValue);
+    }
+
+    @Test
+    public void leaveSerializeCreatedAndLastModifiedAndContent() throws RepositoryException, ParseException
```

Comment:
```suggestion
    public void leaveSerializesCreatedAndLastModifiedAndFileContent() throws RepositoryException, ParseException
```

But I would split this into two tests, one for `created` and `lastModified`, and one for file content.